test/functional/api/v2/discovery_rules_controller_test.rb

```diff
@@ -48,6 +48,14 @@ class Api::V2::DiscoveryRulesControllerTest < ActionController::TestCase
     assert_response :success
   end
 
+  test "should update taxonomy for discovery rule" do
+    rule = FactoryGirl.create(:discovery_rule)
+    organization = FactoryGirl.create(:organization)
+    put :update, { :id => rule.to_param, :discovery_rule => { :organization_ids => [organization.id] } }
+    assert_response :success
+    assert_equal(rule.organization_id, organization.id)
```

Comment:
Yeah, have a typo there... will fix soon

2016-07-21 11:24 GMT+03:00 Lukáš Zapletal <notifications@github.com>:

> In test/functional/api/v2/discovery_rules_controller_test.rb
> <https://github.com/theforeman/foreman_discovery/pull/280#discussion_r71664975>
> :
>
> > @@ -48,6 +48,14 @@ class Api::V2::DiscoveryRulesControllerTest < ActionController::TestCase
> >      assert_response :success
> >    end
> >
> > +  test "should update taxonomy for discovery rule" do
> > +    rule = FactoryGirl.create(:discovery_rule)
> > +    organization = FactoryGirl.create(:organization)
> > +    put :update, { :id => rule.to_param, :discovery_rule => { :organization_ids => [organization.id] } }
> > +    assert_response :success
> > +    assert_equal(rule.organization_id, organization.id)
>
>   1) Error:
> Api::V2::DiscoveryRulesControllerTest#test_0005_should update taxonomy for discovery rule:
> NoMethodError: undefined method `organization_id' for #<DiscoveryRule:0x0055e20df64cb8>
> Did you mean?  organization_ids
>                organization_ids=
>                organization_ids_was
>                organizations
>                organizations=
>     /home/lzap/work/foreman_discovery/test/functional/api/v2/discovery_rules_controller_test.rb:56:in `block in <class:DiscoveryRulesControllerTest>'
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/foreman_discovery/pull/280/files/47fc7d899fd416aae8c8a29cde45ceff72329966#r71664975>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAadr7GV3RBOkn_1a8oUKxMedNlRy9dqks5qXyyngaJpZM4JAC6R>
> .
>